apple/macbook-pro/12-1/default.nix

```diff
@@ -7,6 +7,20 @@
     <nixpkgs/nixos/modules/hardware/network/broadcom-43xx.nix>
```

Comment:
wanted to discuss also this module, I've tried unloading and brcmfmac and loading b43, but it did not work for me. i suspect that it lingers here from older commits.

Reply:
cc @teh who added this.

Reply:
All this module does is `hardware.enableRedistributableFirmware = true;`. Is there any meaningful message in dmesg when doing the modprobe?

Reply:
This is where b43 lives: https://github.com/torvalds/linux/tree/master/drivers/net/wireless/broadcom/b43

Reply:
ah, indeed. so it's not using b43.

Reply:
```console
[  175.994580] cfg80211: Loading compiled-in X.509 certificates for regulatory database
[  175.994858] cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
[  175.994895] platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
[  175.994897] cfg80211: failed to load regulatory.db
[  176.055245] Broadcom 43xx driver loaded [ Features: PNL ]
```

but since this driver is not used, i think it does not matter much.

Reply:
do you think we might put load/unload of module here: <nixpkgs/nixos/modules/hardware/network/broadcom-43xx.nix> ?